constructorio-client/src/test/java/io/constructor/client/ConstructorIOAutocompleteTest.java

```diff
@@ -82,4 +84,27 @@ public void autocompleteShouldReturnAResultWithHiddenFields() throws Exception {
         assertEquals("autocomplete product suggestions exist", response.getSections().get("Products").size(), 5);
         assertEquals("autocomplete result [testField] exists", response.getSections().get("Products").get(0).getData().getMetadata().get("testField"), "hiddenFieldValue");
     }
+
+    @Test
+    public void autocompleteShouldReturnAResultWithOneFilter() throws Exception {
+        ConstructorIO constructor = new ConstructorIO("", apiKey, true, null);
+        UserInfo userInfo = new UserInfo(3, "c62a-2a09-faie");
+        AutocompleteRequest request = new AutocompleteRequest("item1");
+        request.getFilters().put("group_id", Arrays.asList("All"));
+        AutocompleteResponse response = constructor.autocomplete(request, userInfo);
+        assertEquals("autocomplete product suggestions exist", response.getSections().get("Products").size(), 2);
+        assertTrue("autocomplete result id exists", response.getResultId() != null);
```

Comment:
Would it be possible to check if the filters were set correctly in the response?

Reply:
I just added those but I am not sure if it's the right way to do it. I couldn't find any examples in the current tests. I will tag Zubin to make sure they are good